packages/odyssey-storybook/src/components/odyssey-mui/Tooltip/Tooltip.stories.tsx

```diff
@@ -175,18 +175,6 @@ export const StatusWrapper: StoryObj<TooltipProps> = {
   },
 };
 
-export const Disabled: StoryObj<TooltipProps> = {
```

Comment:
Removing this since the story doesn't function (tooltip on the disabled button) and it would require additional documentation for the background provider work.

Reply:
Thanks. I removed this in my branch as well haha